spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :request do
+  describe 'GET #index' do
+    it 'returns http success' do
+      get users_path
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'returns correct response status' do
+      get users_url
+      expect(response).to be_successful
+    end
+
+    it 'renders the index template' do
+      get users_url
+      expect(response).to render_template(:index)
+    end
+
+    it 'includes correct placeholder text in the response body' do
+      get users_url
+      expect(response.body).to include('Here are all the users:')
+    end
+  end
+
+  describe 'GET #index' do

```

Comment:
- [ ] [OPTIONAL] Kindly make sure to pay attention to small details for example here the test should be `'Get #show'`, this may cause confusion in debugging if an error occurred

Reply:
Sure brother @Ismailco.